src/options/options.html

```diff
@@ -14,6 +14,10 @@
                 <label>Open as popup:</label>
                 <input id="gitpod-open-as-popup" type="checkbox"/>
             </div>
+            <div class="row">
```

Comment:
**thought**: Dropping this comment here since this is the moste relevant files changes in this PR. Ideally, in the long run we could move the options dialog (modal) away from the extension settings page into a standalone settings page. This could improve UX when interacting with the browser extension settings but also provide room for more ideas and improvements.